src/components/PropTable.js

```diff
@@ -82,6 +86,7 @@ export default class PropTable extends React.Component {
               <td>{row.propType}</td>
```

Comment:
I think we can get the propType very accurately from docgenInfo. 
AFAIK, currently we don't can't show each and every propTypes.